packages/array-mean/src/__tests__/test.js

```diff
@@ -0,0 +1,15 @@
+import mean from '..';
+
+describe('array-mean', () => {
+    it('should return the mean', () => {
+        expect(mean([])).toEqual(undefined);
+        expect(mean([0])).toEqual(0);
```

Comment:
for exact values, it's better to use `expect(...).toBe(value)`. It uses `===` instead of `toEqual` that uses `==`